typings/components.d.ts

```diff
@@ -104,6 +104,7 @@ declare module '@vue/runtime-core' {
     ElAnchor: typeof import('../packages/element-plus')['ElAnchor']
     ElAnchorLink: typeof import('../packages/element-plus')['ElAnchorLink']
     ElSegmented: typeof import('../packages/element-plus')['ElSegmented']
+    ElMention: typeof import('../packages/element-plus')['ElMention']
```

Comment:
ElMention Component should also be exported from the [global.d.ts](https://github.com/element-plus/element-plus/blob/dev/global.d.ts)

Reply:
ok